db/migrate/20171011154900_fix_remaining_specialist_publisher_first_published_at_dates.rb

```diff
@@ -0,0 +1,19 @@
+class FixRemainingSpecialistPublisherFirstPublishedAtDates < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def up
+    scope = Edition.where(publishing_app: "specialist-publisher")
+                   .where.not(document_type: %w(finder finder_email_signup gone redirect))
```

Comment:
Would it be better to say `where(schema_name: 'specialist_document')`?

Reply:
I get slightly different results using that clause rather than the exclusive one. eg.

```
irb(main):001:0> Edition.where(publishing_app: "specialist-publisher").where.not(document_type: %w(finder finder_email_signup gone redirect)).where("first_published_at > (public_updated_at + interval '1 second')").group(:document_type).count
=> {"cma_case"=>31, "drug_safety_update"=>6, "aaib_report"=>8, "raib_report"=>8, "dfid_research_output"=>95, "esi_fund"=>11, "medical_safety_alert"=>5, "specialist_document"=>1}
irb(main):002:0> Edition.where(publishing_app: "specialist-publisher").where(schema_name: "specialist_document").where("first_published_at > (public_updated_at + interval '1 second')").group(:document_type).count
=> {"cma_case"=>26, "drug_safety_update"=>6, "aaib_report"=>7, "raib_report"=>8, "dfid_research_output"=>95, "esi_fund"=>10, "medical_safety_alert"=>5, "specialist_document"=>1}
```

Reply:
Hmm, strange... looks like a few more editions will be fixed by your query so I'm happy to go with yours.